pkg/config/dynamicconfig.go

```diff
@@ -62,13 +95,25 @@ func NewConfigData(rclient kubernetes.Interface, cmInformer informers.ConfigMapI
 		cmSycned: cmInformer.Informer().HasSynced,
 		log:      log,
 	}
+	cd.restrictDevelopmentUsername = []string{"minikube-user", "kubernetes-admin"}
```

Comment:
Shouldn't the init value be taken from the Flag? Can you update the doc on how should the user configure these flags / configmap? We can make it a separate PR.